services/movie_session.py

```diff
@@ -42,3 +42,11 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider handling the potential `DoesNotExist` exception when deleting the `MovieSession` instance. This will prevent the application from crashing if the session ID does not exist.